vyper/semantics/analysis/local.py

```diff
@@ -381,6 +381,9 @@ def visit_For(self, node):
                     if args[0].value >= args[1].value:
                         raise StructureException("Second value must be > first value", args[1])
 
+                if not type_list:
```

Comment:
is this a stray line from another PR?

Reply:
No. Without this line, valid iterators like `for i in range(max_value(int128), max_value(int128)+2)` would raise an `InvalidType` exception with "Not an iterable type" as the error message at L396 instead, which is misleading.